client/reader/start/card.jsx

```diff
@@ -0,0 +1,31 @@
+// External dependencies
+import React from 'react';
+
+// Internal dependencies
+import Card from 'components/card';
+import SiteIcon from 'components/site-icon';
+import FollowButton from 'reader/follow-button';
+import StartPostPreview from './post-preview';
+
+const StartCard = React.createClass( {
```

Comment:
Agreed on treating them as reusable components, but placing them in `reader/reader-site-card` until we figure out where to place app-components to not mix them with general ui components.